Gemfile

```diff
@@ -68,10 +68,13 @@ gem("rtf")
 gem("xmlrpc")
 
 # Simple versioning
-gem("cure_acts_as_versioned")
+# Use our own fork, which stores enum attrs as integers in the db
+gem("cure_acts_as_versioned",
+    git: "https://github.com/MushroomObserver/acts_as_versioned/")
```

Comment:
Is there a better way to do this?

Reply:
This seems reasonable to me until/unless you can get your fixes committed to the original gem and registered with ruby gems.